packages/blazor-workspace/NimbleBlazor/Components/NimbleTable.razor.cs

```diff
@@ -33,6 +33,9 @@ public partial class NimbleTable<TData> : ComponentBase
     [Parameter]
     public TableRowSelectionMode? SelectionMode { get; set; }
 
+    [Parameter]
+    public bool ActionMenusPreserveSelection { get; set; }
```

Comment:
I believe this should be `bool?`.


```suggestion
    public bool? ActionMenusPreserveSelection { get; set; }
```